python/ql/src/semmle/python/objects/ObjectAPI.qll

```diff
@@ -743,6 +743,11 @@ module ClassValue {
     ClassValue nonetype() {
         result = TBuiltinClassObject(Builtin::special("NoneType"))
     }
+    
+    /** Get the `ClassValue` for the `TypeError` class */
+    ClassValue typeErrorType() {
```

Comment:
I would change this to just `typeError`, for consistency with the other methods.

Reply:
Done!